pom.xml

```diff
@@ -50,15 +50,18 @@
         <revision>1.33.0</revision>
         <changelist>-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/github-plugin</gitHubRepo>
-        <jenkins.version>2.164.3</jenkins.version>
+        <jenkins.version>2.222.4</jenkins.version>
```

Comment:
97% of users are on this version (https://stats.jenkins.io/pluginversions/github.html), and it's the minimum supported for new versions of bom currently.

also see https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/